### DISCUSSION TITLE
Why was docker-compose.yaml name changed to compose.yaml?

### DISCUSSION THREAD

**Discussion by fawzibr** [docker-mailserver/docker-mailserver#3412] [Q&A] [answered]
Before you could just do: 

`docker-compose up -d`
`docker-compose down`

Now you have to do 

`docker-compose -f ./compose.yaml up -d`
`docker-compose -f ./compose.yaml down`

I rename it after downloading it anyway. Just curious, why was it changed?

**Comment by fawzibr** [comment]
Nevermind, it seems compose is now integrated in docker and it uses compose.yaml. I have the old docker and docker-compose in Synology.

**Comment by casperklein** [comment, marked as answer]
Here is the PR / discussion about it: #3295
